Sources/Logging/Logging.swift

```diff
@@ -498,8 +498,16 @@ public struct MultiplexLogHandler: LogHandler {
     }
 }
 
+private struct StandardErrorOutputStream: TextOutputStream {
+    func write(_ string: String) {
+        string.withCString { ptr in
+            _ = fputs(ptr, stderr)
+        }
+    }
+}
+
 /// Ships with the logging module, really boring just prints something using the `print` function
-internal struct StdoutLogHandler: LogHandler {
+internal struct StderrLogHandler: LogHandler {
```

Comment:
this change intentional as part of this PR?

Reply:
omg, definitely not, sorry about that!

Reply:
@tomerd done